packages/core/http/core-http-server-internal/src/cdn_config/cdn_config.ts

```diff
@@ -34,13 +34,14 @@ export class CdnConfig {
   public getCspConfig(): CspAdditionalConfig {
     const host = this.host;
     if (!host) return {};
+    // We exclude `connect_src` from this list as it causes issues for features
+    // like maps that rely on being able to load any external resources.
```

Comment:
One alternative is to set `connect_src` to `https:` _if_ the CDN URL's protocol is also HTTPS. But not sure we want to make that assumption here.

---

Another alternative is to not set any CSP here and rather drive this by setting everything in `csp.*` yml config. Kind of depends on what we think about [this issue](https://github.com/elastic/kibana/issues/177732).